### DISCUSSION TITLE
SCF_UNCONVERGED Message

### DISCUSSION THREAD

**Discussion by tom11si** [jensengroup/molcalc#46] [Q&A] [answered]
Hello,

I'd just like to preface by saying that I appreciate this tool so much! It is greatly furthering my chemistry endeavours :) However, when I try to calculate the quantum values for H-I, it replies with a pop up message of SCF_UNCONVERGED. To my understanding this isn't a chemistry error regarding H-I according to Google? Anyways - just letting you know. Thanks a lot.

**Comment by charnley** [comment, marked as answer]
Hello! This is a quantum chemistry message. It stands for self-consistent field theory and didn't converge the molecular orbitals. If you are getting into computational chemistry, I can recommend the book; https://www.amazon.de/-/en/Jan-H-Jensen/dp/1420075268 from Jan Jensen.

**Reply by tom11si** [reply]
Thank you very much! My apologies for assuming it was an error - I am a highschool student and evidently very inexperienced in this field, haha. I appreciate the clarification highly and thanks for the book recommendation :)!
